R/utils.r

```diff
@@ -57,3 +57,18 @@ find_function_name <- function(fun) {
     }
   }
 }
+
+##' Negative binomial random numbers parametrized
+##' in terms of mean and coefficient of variation
+##' @param n number of samples to draw
+##' @param mn mean of distribution
+##' @param disp dispersion coefficient (var/mean)
+##' @return vector containing the random numbers
+##' @author Flavio Finger
+##' @export
+##' @examples
+##' rnb_mean_disp(n = 5, mn = 4, disp = 2)
+rnb_mean_disp <- function(n, mn, disp) {
```

Comment:
Minor comment - shall we call it `rnbinom_mean_disp` to be more similar to `rnbinom`?